bundles/org.openhab.binding.mqtt/README.md

```diff
@@ -7,7 +7,7 @@ MQTT is a server/client architecture.
 
 A server, also called broker is not provided within this binding.
 You can use any of the freely available MQTT Brokers like [Mosquitto](https://mosquitto.org/)
-or [Moquette](https://moquette-io.github.io/moquette/) or install the included Moquette broker as addon.
+or [Moquette](https://moquette-io.github.io/moquette/) or install the [included Moquette broker](/bundles/org.openhab.io.mqttembeddedbroker/README.md) as addon.
```

Comment:
The correct link on the website would be `https://www.openhab.org/addons/integrations/mqttembeddedbroker/`. I doubt that this will be translated correctly.

Reply:
Are you a 100% sure? I'm no expert in exporting Github documentation to a website but I copied the format from another Openhab doc page on Github. The relative link works in preview. Also I read this blog post: https://github.blog/2016-12-05-relative-links-for-github-pages/ . I think it should work but I am not 100% sure and I think there is no way of testing it before it goes live.

Reply:
No. Can you point me to the openhab doc page that is using this format?

Reply:
The website crawler script would need to convert that link. And the crawler is a dated groovy script that no one is really happy with, I wouldn't trust it. Just use an absolute link.

Reply:
It could have been so beautiful :). I made a new commit with an absolute url. It works for sure now (and might stop working when openhab changes it's domain name :)).